packages/module/patternfly-docs/content/extensions/chatbot/design-guidelines.md

```diff
@@ -222,22 +296,36 @@ There are a few display modes that users can choose when interacting with a Chat
 ![ChatBot in overlay mode.](./img/overlay.svg)
 </div>
 
-2. **Docked:** Anchors the ChatBot to the side of the page content. When docked, the ChatBot window is persistent, and cannot be toggled.
+2. **Docked:** Anchors the ChatBot to the side of the page, on top of the page content. When docked, the ChatBot window is persistent, and cannot be toggled. To minimize the ChatBot, users will need to return to overlay mode. 
 
 <div class="ws-docs-content-img">
 ![ChatBot in docked mode.](./img/docked.svg)
 </div>
 
-3. **Full screen:** Fills the screen with the ChatBot window.
+3. **Embedded:** Places the ChatBot within a product as its own page. An embedded ChatBot could be displayed in the product's navigation menu.
+
+<div class="ws-docs-content-img">
+![ChatBot embedded in a product.](./img/embedded.svg)
+</div>
+
+4. **Full screen:** Fills the screen with the ChatBot window.
 
 <div class="ws-docs-content-img">
 ![ChatBot in full-screen mode.](./img/fullscreen.svg)
 </div>
 
-4. **Embedded:** Places the ChatBot within a product as its own page. An embedded ChatBot could be displayed in the product's navigation menu.
+5. **Inline drawer**: Places the ChatBot to the side of the page, inline with the page content.
 
 <div class="ws-docs-content-img">
-![ChatBot embedded in a product.](./img/embedded.svg)
+![ChatBot in drawer mode.](./img/drawer.svg)
+</div>
+
+### Layouts
+
+ChatBot supports a side-by-side [comparison layout](/patternfly-ai/chatbot/overview/demo#comparing-chatbots), which allows users to compare 2 chats at once. This lets users understand how different models could respond to the same prompt, you can add multiple ChatBots within the same window. 
```

Comment:
```suggestion
ChatBot supports a side-by-side [comparison layout](/patternfly-ai/chatbot/overview/demo#comparing-chatbots), which allows users to compare 2 chats at once. This lets users understand how different models could respond to the same prompt. You can add multiple ChatBots within the same window. 
```

Maybe a typo? Not sure.

Reply:
Oh yes definitely something like that 😂 I think I lost my train of thought/didn't finish the sentence lol. ty!